<launch>
    <include file="$(find cob_arm_navigation_but)/launch/cob3_planning_environment.launch" />
    <node pkg="arm_kinematics_constraint_aware" type="arm_kinematics_constraint_aware" name="cob3_arm_kinematics">
        <param name="group" type="string" value="arm" />
        <param name="arm/root_name" type="string" value="base_link" />
        <param name="arm/tip_name" type="string" value="arm_7_link" />
        <param name="kinematics_solver" type="string" value="arm_kinematics_constraint_aware/KDLArmKinematicsPlugin" />
    </node>
</launch>
